Automattic/jetpack
# Use dynamic Jetpack logos on JITMs

#### Changes proposed in this Pull Request:
Depending on the different areas of wp-admin, show just the green Jetpack logo in a JITM when a larger one is already displayed, like in the Jetpack dashboard. Everywhere else, a larger logo (with the "Jetpack" text) should be displayed, provided a Jetpack logo doesn't already exist in the area.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
This modifies some graphical aspects of the Jetpack experience (JITMs) to improve brand placement. Please refer to this Asana task for more info: 
1165973696403626-as-1165991880476585

#### Testing instructions:
1. Create a test JITM that should be displayed everywhere.
2. Refer to these screenshots: 
![jetpack-logo-jitm-use](https://user-images.githubusercontent.com/2166135/77387825-2b16d180-6d6d-11ea-8538-d499ef7dc3fa.jpg)

#### Proposed changelog entry for your changes:
Use dynamic Jetpack logos on JITMs
